build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -175,7 +179,7 @@ function updateReporters(config) {
   if (isTravisPushBuild() && JSON_REPORT_TEST_TYPES.has(config.testType)) {
     config.reporters.push('json-result');
     config.jsonResultReporter = {
-      outputFile: `results_${config.testType}.json`,
+      outputFile: resultFilename(config.testType),
```

Comment:
Nice job pulling this out into a helper!

Reply:
Thanks! I realized if someone changed the filename here, things would get messed up on the other file, which would've been unintuitive.